packages/server/src/util.js

```diff
@@ -18,7 +18,7 @@ export const clearModuleCache = moduleName => {
 }
 
 export const smartRequire = modulePath => {
-  if (process.env.NODE_ENV !== 'production') {
+  if (process.env.NODE_ENV !== 'production' || process.env.LOADABLE_KEEP_CACHE !== '1') {
```

Comment:
According to the ticket description, this is an __incorrect__ way to fix it, as `process.env.NODE_ENV !== 'production'` is still true, and the only change here - you are adding an ability to cleanup module cache in production is as well.

Pretty sure you want something else